docs/mmteb/points.md

```diff
@@ -82,4 +82,7 @@ Please also add your first name and last name are as you want them to appear in
 | shreeya-dhakal            | Shreeya     | Dhakal     | ssdhakal57@gmail.com      |                      | Individual Contributor                                                   |
 | dipam7 | Dipam | Vasani | dipam44@gmail.com | ~Dipam_Vasani1 | Individual Contributor                                                  |
 | jankounchained    | Jan        | Kostkan    | jan.kostkan@cas.au.dk | ~Jan_Kostkan1        | Aarhus University, Denmark                            |
-
+| rasdani           | Daniel     | Auras      | daniel@ellamind.com          |                        | ellamind, Germany                                   |
```

Comment:
Seems like you are missing the openreview ID

Reply:
None of us has any 😄
Is this necessary?

Reply:
It is required for us to add you as a co-author to the publication

Reply:
added our Openreview IDs